tools/push-docker-image.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+version=$1
+
+cd ${version} || exit 1
+
+docker build -t newrelic/php-daemon:latest -t newrelic/php-daemon:${version} .
+
+docker login --username=newrelicphp --password ${DOCKER_HUB_PASSWORD}
+
+docker push newrelic/php-daemon
```

Comment:
I also mentioned this in the C daemon PR: I wonder whether it's possible to just push dedicated tags here?

Reply:
I don't know that we can do that. I can clear all the images and tags at the beginning of the script though.

Reply:
👍